runconfig/config_windows.go

```diff
@@ -0,0 +1,13 @@
+package runconfig
+
+// ContainerConfigWrapper is a Config wrapper that hold the container Config (portable)
+// and the corresponding HostConfig (non-portable).
+type ContainerConfigWrapper struct {
+	*Config
+	HostConfig *HostConfig `json:"HostConfig,omitempty"`
```

Comment:
What is interesting, and I really can't explain. If I lowercase it, the tests do not pass locally Windows to Linux. Yet if I upper case it (also causing it to be exported, even though never used), the tests DO pass locally.